matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/migration/MigrateSessionTo008.kt

```diff
@@ -25,11 +25,11 @@ internal class MigrateSessionTo008(realm: DynamicRealm) : RealmMigrator(realm, 8
 
     override fun doMigrate(realm: DynamicRealm) {
         val editionOfEventSchema = realm.schema.create("EditionOfEvent")
-                .addField(EditionOfEventFields.CONTENT, String::class.java)
+                .addField("content"/**EditionOfEventFields.CONTENT*/, String::class.java)
```

Comment:
Maybe we should add a comment about this commented code? I guess you wanted to keep the previous fields in comments for future reference?

Reply:
Kept it to show that it was from an added then deleted class, ok to remove though